### PR TITLE
EDSC-2058: Splits out job processings methods into standalone jobs

### PR DESCRIPTION
**Pull request by abbottry**

### CODE REVIEW

File: app/controllers/concerns/granule_utils.rb [hunk 1]
```diff
@@ -77,6 +77,7 @@ def fetch_ous_response(options)
     ous_params['bounding_box'] = project_params['bounding_box'] if project_params.key?('bounding_box')
     ous_params['temporal']     = project_params['temporal'] if project_params.key?('temporal')
 
+    puts ous_params
```
Comment (macrouch) [inline app/controllers/concerns/granule_utils.rb]: puts

File: app/controllers/oauth_tokens_controller.rb [hunk 2]
```diff
@@ -24,25 +24,53 @@ def refresh_token
     if json
       render json: { tokenExpiresIn: script_session_expires_in }
     else
-      render json: nil, status: 401
+      render json: nil, status: :unauthorized
     end
   end
 
-  protected
+  def store_user_data(oauth_response)
```
Comment (macrouch) [inline app/controllers/oauth_tokens_controller.rb]: Why did you remove `protected`?

Comment (abbottry) [reply app/controllers/oauth_tokens_controller.rb]: No reason, must have been removed when I removed other stuff in that file. I'll replace it.

File: app/controllers/concerns/granule_utils.rb (hunk 1 above)
Comment (abbottry) [reply app/controllers/concerns/granule_utils.rb]: Last minute debug -- good catch.

File: app/models/order.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+class Order < ActiveRecord::Base
+  belongs_to :retrieval_collection
+  belongs_to :orderable, polymorphic: true
```
Comment (abbottry) [inline app/models/order.rb]: Remove
